contents/handbook/growth/marketing/seo-guide.md

```diff
@@ -0,0 +1,110 @@
+---
+title: SEO best practices
```

Comment:
This should be an article you publish!

Reply:
Yeah, this is a little too off the top of my head, but it's in the back of my head for the near-ish future.